src/rulesets/oas/index.json

```diff
@@ -515,7 +512,7 @@
       "message": "{{error}}",
       "recommended": true,
       "formats": ["oas2"],
-      "severity": "error",
```

Comment:
@philsturgeon I do think "error" would be nicer but I was more focused on making the tests pass.

Proposal: Unless this is blocking, maybe log a tech debt issue and switch back to "error" in another PR?